dir.targets

```diff
@@ -36,7 +36,7 @@
     <!-- Restore build tools -->
     <Exec
       StandardOutputImportance="Low"
-      Command="&quot;$(NuGetToolPath)&quot; install &quot; $(SourceDir).nuget\packages.config &quot;  -o &quot; $(ToolsDir) &quot; $(NuGetConfigCommandLine)" />
+      Command="&quot;$(NuGetToolPath)&quot; install &quot;$(PackagesGlobalConfig)&quot;  -o &quot;$(ToolsDir)&quot; $(NuGetConfigCommandLine)" />
```

Comment:
afaik the space after `$(ToolsDir)` is necessary so the trailing backslash doesn't escape the quote on Windows. Using `$(ToolsDir.TrimEnd('\'))` should be equivalent.